### PR TITLE
Add the "google_compute_address" datasource

### PR DESCRIPTION
**Pull request by eilgin**
As said in #712, it could be useful to have that kind of datasource to better handle IP creation/updates, e.g. for use in other providers such as cloudfare.

### CODE REVIEW

**Comment by eilgin** [comment]
Acceptance test result:
```bash
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccDataSourceGoogleAddress -timeout 120m
?       github.com/terraform-providers/terraform-provider-google        [no test files]
=== RUN   TestAccDataSourceGoogleAddress
--- PASS: TestAccDataSourceGoogleAddress (26.30s)
PASS
ok      github.com/terraform-providers/terraform-provider-google/google 26.321s
testing: warning: no tests to run
PASS
ok      github.com/terraform-providers/terraform-provider-google/scripts        0.030s [no tests to run]
```

File: google/data_source_google_compute_address_test.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccDataSourceGoogleAddress(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
```
Comment (rosbo) [inline google/data_source_google_compute_address_test.go]: Can you add the CheckDestroy: 

`CheckDestroy: testAccCheckComputeAddressDestroy,`

File: website/docs/d/datasource_compute_address.html.markdown [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
```
Comment (rosbo) [inline website/docs/d/datasource_compute_address.html.markdown]: you also need to update website/google.erb if you want your documentation page to show up in the side menu

**Comment by eilgin** [comment]
@rosbo i've applied the changes you've requested ;)

**Comment by rosbo** [comment]
Thank you Thomas. Merging :)

**Pull request merged**